app/Console/Commands/Setup/SetupSyncPteroCommand.php

```diff
@@ -59,6 +60,16 @@ public function handle()
         Egg::syncEggs();
         $bar->advance();
 
+        $this->newLine();
+        $this->line('Syncing Servers..this might take a while.');
+
+        Server::inRandomOrder()->chunk(15, function($servers) {
```

Comment:
Please add docblocks for the types :)  Servers[]

Reply:
you and your docblocks xD